src/components/ProjectSettingsTabs/utils.ts

```diff
@@ -1,5 +1,45 @@
+import { SpruceFormProps } from "components/SpruceForm";
 import { ProjectSettingsTabRoutes } from "constants/routes";
 
 export interface TabProps {
   tab: ProjectSettingsTabRoutes;
 }
+
+const insertIf = (condition, ...elements) => (condition ? elements : []);
+
+/**
+ * Generate the options for a radio box group that conditionally includes a third option to "Default to repo".
+ *
+ * @param {options} string array representing the primary two radio box labels.
+ * @param {field} value upon which the default box should be conditionally shown.
+ * @param {invert} specify whether or not the field represents an "inverted" value; i.e. the option indicates a field is disabled rather than enabled.
```

Comment:
the term invert confuses me a bit, can you please explain a bit more? how do we know what value it holds?